### ISSUE TITLE
translation of 'blocks are flying in the wire for the whole server'

### ISSUE THREAD

**Issue by BrunoMine** [minetest/minetest#4340]
About the sentence:
_"How many blocks are flying in the wire simultaneously for the whole server."_
[Comments on weblate](https://hosted.weblate.org/translate/minetest/minetest/pt_BR/?sid=3cb09718-4ec3-11e6-addf-5254000d9bae&offset=4#comments)
I think that i don't understand.
This means:
_"How many blocks simultaneously pass to all the server map."_
If I'm wrong, explain to me.

**Comment by t4im** [comment]
Can be rephrased in https://github.com/minetest/minetest/blob/master/builtin/settingtypes.txt#L661
```ini
#    How many blocks are flying in the wire simultaneously per client.
max_simultaneous_block_sends_per_client (Maximum simultaneously blocks send per client) int 10

#    How many blocks are flying in the wire simultaneously for the whole server.
max_simultaneous_block_sends_server_total (Maximum simultaneously bocks send total) int 40
```

**Comment by BrunoMine** [comment]
This helps a lot.

**Comment by t4im** [comment]
Why are you closing the issue?

The text still needs to be rephrased. It is clearly confusing.

**Comment by BrunoMine** [comment]
I think that a good rephrase for those sentences are:

How many blocks are flying in the wire simultaneously per client.
 ^---> _How many blocks can be sent simultaneously per client._

How many blocks are flying in the wire simultaneously for the whole server.
 ^---> _How many blocks can be sent simultaneously to the entire server._

The requirements for a good sentence is:

- simplified semantic
- usual terms
- well targeted idea

Maybe I'm wrong, but that's what I got so far.

**Comment by paramat** [comment]
(Maximum simultaneously blocks send per client)
(Maximum simultaneously bocks send total)

Wow these need rewording.

**Comment by 0-afflatus** [comment]
Yep. I find these sentences hard to understand and I'm a native English speaker.

**Comment by red-001** [comment]
fixed this in #4475

**Comment by paramat** [comment]
#4475 is merged.
